src/crypto/clu_encrypt.c

```diff
@@ -1,8 +1,8 @@
 /* clu_encrypt.c
  *
- * Copyright (C) 2006-2020 wolfSSL Inc.
+ * Copyright (C) 2006-2021 wolfSSL Inc.
```

Comment:
Longer term, it would be nice to merge clu_encrypt and clu_decrypt into one file. The functionality is very similar. Only the data flow is different.

Reply:
was hoping that eventually EVP would be used and clu_encrypt/clu_decrypt would go away. They are kept because of use with the existing Camellia algorithm. EVP is the way to go with the future additions though because of padding interop with OpenSSL.